libs/utils/src/pageserver_feedback.rs

```diff
@@ -29,12 +29,10 @@ pub struct PageserverFeedback {
     // Serialize with RFC3339 format.
     #[serde(with = "serde_systemtime")]
     pub replytime: SystemTime,
+    /// Used to track feedbacks from different shards. Always zero for unsharded tenants.
+    pub shard_number: u32,
```

Comment:
This could be a `ShardNumber`?  Or were you intentionally using a bare type to make it obvious for the serialization?

Reply:
>ShardNumber is defined in pageserver_api crate, which is not a dependency of utils.

Hmm, this makes me think that maybe PageserverFeedback should live somewhere other than utils -- it's not a utility.  But that's not this PR's fault.

>Also, PageserverFeedback is a binary protocol, it feels safer to not depend on an external type that can change and break the compatibility.

Fair enough